FilterViewController.m

```diff
@@ -125,28 +189,26 @@ - (void) createModalWithType:(CRUCellViewInteractionType)cellType section:(MHCol
     
 }
 
-- (void)saveChangesForCurrentSection{
+    }
```

Comment:
These brackets are confusing. I can't tell if they are matched with previous brackets. Is all of this code inside the buttonTapped method?